packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -729,23 +733,15 @@ abstract class BaseFlutterTask extends DefaultTask {
             args "--depfile", "${intermediateDir}/flutter_build.d"
             args "--output", "${intermediateDir}"
             args "-dTargetFile=${targetPath}"
-            args "-dTargetPlatform=${targetPlatform}"
+            args "-dTargetPlatform=TargetPlatform.android"
```

Comment:
If the target platform is extracted from the target name, then why is this needed?

Reply:
KernelSnapshot requires a target platform per your fuchsia changes. We could loosen that restriction a bit by making a missing platform default to non-fuchsia

Reply:
We'd also still need it if we wanted to do platform specific assets at some point